package.json

```diff
@@ -44,21 +44,23 @@
     "test.watch": "yarn test --watch"
   },
   "dependencies": {
-    "@stoplight/json": "^3.4.0",
-    "@stoplight/path": "^1.3.1",
-    "@stoplight/types": "^11.6.0",
-    "@types/urijs": "^1.19.9",
-    "dependency-graph": "~0.8.0",
-    "fast-memoize": "^2.5.1",
-    "immer": "^5.3.2",
-    "lodash": "^4.17.15",
```

Comment:
Just in case Lodash has some security vulnerability.
We only use 2 functions, thus let's mitigate the potential risk.